msi/GitProduct.wxs

```diff
@@ -295,9 +315,12 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureFsCache" Before="ExecPostInstallBat">
+      <Custom Action="ExecConfigureFsCache" Before="ExecConfigureGcm">
         <![CDATA[REMOVE <> "ALL" AND ENABLEFSCACHE = "true"]]>
       </Custom>
+      <Custom Action="ExecConfigureGcm" Before="ExecPostInstallBat">
+        <![CDATA[REMOVE <> "ALL" AND WIX_IS_NETFRAMEWORK_451_OR_LATER_INSTALLED AND ENABLEGCM]]>
```

Comment:
For some reason it only occurred to me *now* that this `REMOVE <> "ALL"` might not be enough: what if the user modifies their installation and specifically unchecks Git Credential Manager? Wouldn't we need another `REMOVE <> "..."`?